src/tag/tag.stories.ts

```diff
@@ -2,44 +2,39 @@ import { storiesOf, moduleMetadata } from "@storybook/angular";
 import { withKnobs } from "@storybook/addon-knobs/angular";
 
 import { TagModule } from "../tag/tag.module";
-import { ExperimentalComponenent } from "../../.storybook/experimental.component";
-import { ExperimentalModule } from "../experimental.module";
 
 storiesOf("Tag", module)
 	.addDecorator(
 		moduleMetadata({
-			declarations: [
-				ExperimentalComponenent
-			],
 			imports: [
-				TagModule,
-				ExperimentalModule
+				TagModule
 			]
 		})
 	)
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
 		template: `
-			<app-experimental-component [(isExperimental)]="experimental"></app-experimental-component>
-			<ibm-tag [type]="(!experimental ? 'ibm' : 'warm-gray')" class="foo {{test}}">{{ !experimental ? "IBM" : "Warm-gray" }}</ibm-tag>
+			<ibm-tag type="warm-gray">Warm-gray</ibm-tag>
 			<br><br>
-			<ibm-tag [type]="(!experimental ? 'beta' : 'red')">{{ !experimental ?  "Beta" : "Red" }}</ibm-tag>
+			<ibm-tag type="red">Red</ibm-tag>
```

Comment:
Tag types don't have some kind of meaning to them? In case the color changes...

Reply:
I'm not sure I understand ... they're just tags, they shouldn't have inherit meaning (that's the applications job)

If the color changes, the class also changes, so the story will _also_ change